charts/rancher-eks-operator/1.0.600/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.6-alpha1
```

Comment:
Do older versions need a rancher max version?

Reply:
It picks the latest one, so not really. cc @rmweir